sdk/devcenter/azure-developer-devcenter/README.md

```diff
@@ -1,156 +1,28 @@
 
-Use the package for Azure DevCenter to:
-> Create, access, manage, and delete Dev Box resources
-> Create, deploy, manage, and delete Environment resources
+## _Disclaimer_
 
-## Getting started
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to https://github.com/Azure/azure-sdk-for-python/issues/20691_
 
-### Installating the package
+# Usage
 
-```bash
-python -m pip install azure-developer-devcenter
-```
 
-#### Prequisites
+To learn how to use this package, see the [quickstart guide](https://aka.ms/azsdk/python/mgmt)
+ 
+For docs and references, see [Python SDK References](https://docs.microsoft.com/python/api/overview/azure/)
+Code samples for this package can be found at [MyService Management](https://docs.microsoft.com/samples/browse/?languages=python&term=Getting%20started%20-%20Managing&terms=Getting%20started%20-%20Managing) on docs.microsoft.com.
+Additional code samples for different Azure services are available at [Samples Repo](https://aka.ms/azsdk/python/mgmt/samples)
 
-- Python 3.7 or later is required to use this package.
-- You need an [Azure subscription][azure_sub] to use this package.
-- You must have [configured](https://learn.microsoft.com/azure/dev-box/quickstart-configure-dev-box-service) a DevCenter, Project, Network Connection, Dev Box Definition, and Pool before you can create Dev Boxes 
-- You must have [configured](https://learn.microsoft.com/azure/deployment-environments/) a DevCenter, Project, Catalog, and Environment Type before you can create Environments
 
-#### Create with an Azure Active Directory Credential
-To use an [Azure Active Directory (AAD) token credential][authenticate_with_token],
-provide an instance of the desired credential type obtained from the
-[azure-identity][azure_identity_credentials] library.
+# Provide Feedback
 
-To authenticate with AAD, you must first [pip][pip] install [`azure-identity`][azure_identity_pip]
+If you encounter any bugs or have suggestions, please file an issue in the
+[Issues](https://github.com/Azure/azure-sdk-for-python/issues)
+section of the project. 
 
-After setup, you can choose which type of [credential][azure_identity_credentials] from azure.identity to use.
-As an example, [DefaultAzureCredential][default_azure_credential] can be used to authenticate the client:
 
-Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
-`AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`
-
-Use the returned token credential to authenticate the client:
-
-```python
->>> import os
->>> from azure.developer.devcenter import DevCenterClient
->>> from azure.identity import DefaultAzureCredential
->>> tenant_id = os.environ['AZURE_TENANT_ID']
->>> client = DevCenterClient(tenant_id=tenant_id, dev_center="my_dev_center", credential=DefaultAzureCredential())
-```
-
-## Examples
```

Comment:
May I know whey these examples are removed?

Reply:
Think this is what @sebrenna is referring to -- the AutorestCI bot keeps adding a commit titled "Packaging update of azure-developer-devcenter" which breaks everything and removes our existing work. How can we disable this bot?

Reply:
Fixed after disabling push from AutorestCI.